### ISSUE TITLE
Allow Aria Label Configurability to improve Accessibility

### ISSUE THREAD

**Issue by dimercurio** [GetStream/stream-chat-react#1994]
**Motivation**
With accessibility being more of a focus lately, we have been looking to improve areas that are within our control in regards to GetStream. We have noticed there are a few concerns we'd like to address that are currently out of our hands as GetStream is hard-coding values/not allowing configurability in some of these areas. 

One thing we find ourselves needing to do is set the `aria-label` for things such as the Emoji and Upload buttons. Although you are able to define custom components for these icons, the parent contains hard-coded labels that can only be addressed by creating an entirely new MessageInput component. 
![HardCodedAriaLabels](https://user-images.githubusercontent.com/37486021/231849211-d9c1bdc6-473b-467e-8a6c-aed7ed405852.png)

 
**Proposed solution**
This could just be an object that is taken in as a prop into Channel, with each key representing the label value you'd like to overwrite (ex. emojiPickerAriaLabel). This would allow us to define custom aria labels depending on user locale.

**Acceptance Criteria**
Please allow for configurability of the aria-labels for the controls/icons within the MessageInput.

**Comment by MartinCupela** [comment]
@dimercurio thank you for raising the request. Similar request came with #1931 . We realize the space for improvement. Another way how the labels could be customizable would be through i18n. I believe that would be even a cleaner solution. What do you think?

**Comment by dimercurio** [comment]
> 

Yes I believe that would be an even better solution for situations like this where aria-labels need to be customized per translation. Perhaps this could be something Stream supports by default. 


Something else we've come across is we've needed the ability to set `aria-expanded` when something like the EmojiPicker is opened. 

![Screen Shot 2023-04-28 at 7 46 33 AM](https://user-images.githubusercontent.com/37486021/235186240-9bfcd518-eb60-4302-adfd-fa051e8e2bc6.png)

The reason I suggest having some sort of a configuration object designated for aria attributes is because it would allow developers to have total control of the properties they want and what their values should be. All we would need from Stream's side is an update to these elements to allow for spreading of these properties to these parent elements, such as the 'emoji picker' button.
